tools/build_tar/buildtar.go

```diff
@@ -276,6 +278,7 @@ func (f *tarFile) addLink(symlink, target string) error {
 		Name:     symlink,
 		Typeflag: tar.TypeSymlink,
 		Linkname: target,
+		ModTime:  mtime,
```

Comment:
I think you can use `f.meta.mtime` if you want to avoid passing it as an argument

Reply:
Done